### ISSUE TITLE
/usr/bin/wget is not created when both wget-ssl and wget-nossl are selected

### ISSUE THREAD

**Issue by 1plus-1** [openwrt/openwrt#17392]
### Describe the bug

/usr/bin/wget is not created when both wget-ssl and wget-nossl are selected. Thus 'apk update' is not working.

Now wget-ssl and wget-nossl have the same alternative priority(300).
https://github.com/openwrt/packages/blob/ca9c3d1baea38f0e5cedae7b74cfe8ef15a3ec83/net/wget/Makefile#L52

When both are selected, function update_alternatives() in function.sh would not create the link due to the best_prio is alwasy equal or larger than pkg_prio.
https://github.com/openwrt/openwrt/blob/d6c5805db37d42d29e190b7f2dc49ed8e4545e00/package/base-files/files/lib/functions.sh#L327

Update wget-ssl and wget-nossl with different priority can fix the problem(probably should be done). But to make it more robust an update to the function.sh script is recommanded. Because the problem could happen for any link has multiple alternatives with same best priority.

The suggest fix would be changing it to '-le'. With this change the source from late built package would overwrite previous one. 
`if [ "$best_prio" -le "$pkg_prio" ]; then`

### OpenWrt version

d1e9c50d06a8cb618cb85ab489cbcccaec220636

### OpenWrt release

SNAPSHOT

### OpenWrt target/subtarget

mediatek/filogic

### Device

Openwrt One

### Image kind

Self-built image

### Terms

- [X] I am reporting an issue for OpenWrt, not an unsupported fork.

**Comment by 1plus-1** [comment]
Yes. It's from mainline 3 days ago: https://github.com/openwrt/openwrt/commit/d1e9c50d06a8cb618cb85ab489cbcccaec220636

**Comment by 1plus-1** [comment]
A ticket to request updating the priority is created: https://github.com/openwrt/packages/issues/25625

**Comment by Neustradamus** [comment]
Linked to:
- https://github.com/openwrt/packages/pull/25633
- https://github.com/openwrt/packages/issues/25625
- https://github.com/openwrt/openwrt/issues/17385
- https://github.com/openwrt/openwrt/issues/17392

**Comment by robimarko** [comment]
Fixed in: https://github.com/openwrt/packages/issues/25625
